components/script/dom/element.rs

```diff
@@ -2974,3 +2991,53 @@ pub fn cors_setting_for_element(element: &Element) -> Option<CorsSettings> {
         }
     })
 }
+
+pub fn lang_check(css_lang: &Box<str>, element_lang: String) -> bool {
+    let css_lang_tags: Vec<&str> = css_lang.split('\x2d').collect();
```

Comment:
Looks wrong. The CSS lang attribute is a language priority list, which contains language ranges. @SimonSapin would that be a correct interpretation?